apps/api/tests/Animals.spec.ts

```diff
@@ -0,0 +1,201 @@
+    const PetList = await apiContext.get(`${url}/animalls`);
+    expect(PetList.status()).toBe(404);
+  });
+
+  //GET First Pet Pass
+  test("GET animal with id:1 should return the first pet of the list with the correct data", async () => {
+    const firstPet = await apiContext.get(`${url}/animals/1`);
+    const responseBody = await firstPet.json();
+    expect(firstPet).toBeOK();
+    const expectedData = AnimalsListDataSet[0];
+    expect(responseBody).toEqual(expectedData);
+  });
+
+  //GET First Pet Fails
+  test("GET animal with id:1 using the incorrect resource should return 404", async () => {
+    const firstPet = await apiContext.get(`${url}/animalls/1`);
+    expect(firstPet.status()).toBe(404);
+  });
+
+  test("GET animal with a non-existent id should return 404", async () => {
+    const firstPet = await apiContext.get(`${url}/animals/45`);
+    expect(firstPet.status()).toBe(404);
+  });
+
+  test("GET animal with an invalid id should return 400", async () => {
+    const firstPet = await apiContext.get(`${url}/animals/ñ`);
+    expect(firstPet.status()).toBe(400);
+  });
+
+  //POST Add New Pet Pass
+  test("POST usign the correct data should add a new pet", async () => {
+    const postPetRequest = await apiContext.post(`${url}/animals`, {
+      data: NewPetData
+    });
+    expect(postPetRequest).toBeOK();
+    const postPetRequestData = await postPetRequest.json();
+
+    // GET Request to compare with
+    const getNewPetResponse = await apiContext.get(`${url}/animals/${postPetRequestData.id}`);
+    const getNewPetResponseData = await getNewPetResponse.json();
+    const expectedData = {
+      ...NewPetData,
+      id: postPetRequestData.id,
+      birthdate: postPetRequestData.birthdate
+    }
+    expect(getNewPetResponseData).toEqual(expectedData);
+  });
+
+  //POST Add New Pet Fails
+  test("POST usign the incorrect data should return 400", async () => {
+    const postPetRequest = await apiContext.post(`${url}/animals`, {
+      data: NewPetDataFail
+    });
+    expect(postPetRequest.status()).toBe(400);
+  });
+
+  test("POST usign invalid parameters should return 400", async () => {
+    const postPetRequest = await apiContext.post(`${url}/animals`, {
+      data: NewPetDataSetFailWrongParameter
+    });
+    expect(postPetRequest.status()).toBe(400);
+  });
+
+  test("POST usign the incorrect resource should return 404", async () => {
+    const postPetRequest = await apiContext.post(`${url}/animalls`, {
+      data: NewPetData
+    });
+    expect(postPetRequest.status()).toBe(404);
+  });
+
+  test("POST usign an empty data set should return 400", async () => {
+    const postPetRequest = await apiContext.post(`${url}/animals`, {
+      data: NewPetDataFailEmpty
+    });
+    expect(postPetRequest.status()).toBe(400);
+  });
+
+  //PUT (Update) Pets Pass
+  test("PUT using the correct data should update a pet", async () => {
+    const PutUpdatedPet = await apiContext.put(`${url}/animals/2`,
```

Comment:
```suggestion
    const putUpdatedPet = await apiContext.put(`${url}/animals/2`,
```
Try to keep consistency on the variable names convention. We are using [lowerCamerCase](https://wiki.c2.com/?LowerCamelCase) for this project.

Reply:
Done, thanks!